src/components/Jobs/JobOverviewTab.tsx

```diff
@@ -0,0 +1,85 @@
+import { Anchor, Badge, Card, Text } from '@mantine/core';
+import { HiOutlineDocumentDownload } from 'react-icons/hi';
```

Comment:
Let's try to stick with some library for icons for consistency. I will message you and the other person about this.